build.gradle

```diff
@@ -267,7 +267,7 @@ subprojects { subproject ->
         testCompile "io.rest-assured:rest-assured:${restAssuredVersion}"
         testCompile group: 'org.powermock', name: 'powermock-api-mockito2', version: powermockVersion
         testCompile group: 'org.powermock', name: 'powermock-module-junit4', version: powermockVersion
-        testCompile group: 'uk.gov.hmcts', name: 'befta-fw', version: '5.13.0'
+        testCompile group: 'uk.gov.hmcts', name: 'befta-fw', version: '5.15.1-RDM-9825'
```

Comment:
is this the correct version of the fw you want to merge into develop branch?

Reply:
No, it's not. I'm merging with the latest stable release, 5.14.1.